### PR TITLE
refactor: use giraffe single stat graph in UI

### PR DESCRIPTION
**Pull request by ChitlangeSahas**
We want to use the giraffe's component instead of the UI's old code. These changes are behind the `useGiraffeGraphs` feature flag.

Giraffe's single stat component renders in UI: 

Screen Shot 2021-03-16 at 12 04 27 PM

### CODE REVIEW

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 43 | :red_circle:
accessibility | 87 | :green_circle:
best-practices | 87 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :question:

**Comment by TCL735** [comment]
This is a good start.

But what about the position of the Single Stat relative to the graph it is overlaying? Giraffe added this capability. The Single Stat can be moved around on the screen. This might be a separate mini-feature and story. Just wanted to let you know this is possible.

**Comment by TCL735** [comment]
@ChitlangeSahas Can you take a screenshot of this working locally and add it to the PR description above? Thanks.

**Comment by ChitlangeSahas** [comment]
> @ChitlangeSahas Can you take a screenshot of this working locally and add it to the PR description above? Thanks.

Yes absolutely!

**Comment by TCL735** [comment]
Per @kristinarobinson a feature flag is required before we make changes here. Please hang onto this until a feature flag is added.

**Review by TCL735 — CHANGES_REQUESTED** [review]
Per @kristinarobinson a feature flag is required before we make changes here. Please hang onto this until a feature flag is added.

File: src/visualization/types/SingleStat/view.tsx [hunk 1]
```diff
@@ -19,54 +24,91 @@ interface Props extends VisualizationProps {
...
-        const {
-          bgColor: backgroundColor,
-          textColor,
-        } = generateThresholdsListHexs({
-          colors,
-          lastValue: latestValue,
-          cellType: 'single-stat',
-        })
+  if (isFlagEnabled('useGiraffeGraphs')) {
+    const latestValues = getLatestValues(result.table)
```
Comment (TCL735) [inline src/visualization/types/SingleStat/view.tsx:28]: `getLatestValues` should come from Giraffe. We should export it and release another version of Giraffe and consume it here.

Comment (TCL735) [reply src/visualization/types/SingleStat/view.tsx:28]: Although not needed here, we should also take the opportunity to also export `formatStatValue`

**Review by TCL735 — CHANGES_REQUESTED** [review]

File: src/visualization/types/SingleStat/view.tsx (hunk 1 above)
Comment (ChitlangeSahas) [reply src/visualization/types/SingleStat/view.tsx:28]: https://github.com/influxdata/giraffe/pull/502

File: src/visualization/types/SingleStat/view.tsx [hunk 2]
```diff
@@ -2,15 +2,22 @@
...
 import {SingleStatViewProperties} from 'src/types/dashboards'
 import {VisualizationProps} from 'src/visualization'
 
+import {
+  Config,
+  Plot,
+  getLatestValues,
+  formatStatValue,
+} from '@influxdata/giraffe'
+import LatestValueTransform from '../../components/LatestValueTransform'
```
Comment (TCL735) [inline src/visualization/types/SingleStat/view.tsx:17]: One more nitpick and this PR will be good to go. This import should not be changed to use relative paths. Can you please restore the original line that was deleted above under "// Utils" ?

**Review by TCL735 — CHANGES_REQUESTED** [review]

File: src/visualization/types/SingleStat/view.tsx (hunk 2 above)
Comment (ChitlangeSahas) [reply src/visualization/types/SingleStat/view.tsx]: Yes, just did, sorry about that, somehow slipped.

**Review by hoorayimhelping — APPROVED** [review]

**Review by TCL735 — APPROVED** [review]

**Pull request merged**